session/session.go

```diff
@@ -2422,7 +2422,7 @@ func (s *session) InitTxnWithStartTS(startTS uint64) error {
 	}
 
 	// no need to get txn from txnFutureCh since txn should init with startTs
-	txn, err := s.store.BeginWithStartTS(oracle.GlobalTxnScope, startTS)
```

Comment:
This would let `PointGet` always use `global` as txnScope.

Reply:
Maybe because PointGet always use max uint64 as start ts, so it doesn't matter.